src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -4259,12 +4265,112 @@ private static BoundExpression SkipReferenceConversions(BoundExpression possibly
             }
         }
 
-        public override BoundNode? VisitConditionalAccess(BoundConditionalAccess node)
+        // PROTOTYPE(improved-definite-assignment):
+        // this is not used outside 'VisitPossibleConditionalAccess' yet, but is expected to be used
+        // when refactoring 'VisitBinaryOperatorChildren'
+        private bool TryVisitConditionalAccess(BoundExpression node, out PossiblyConditionalState stateWhenNotNull)
```

Comment:
nit: Please add xml docs for this and following method to make contract explicit

Reply:
Done.